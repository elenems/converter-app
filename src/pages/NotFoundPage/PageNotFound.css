#pageNotFound {
  display: flex;
  height: 100vh;
  align-items: center;
  justify-content: center;
}

#pageNotFound > div {
  text-align: center;
  padding: 100px;
  border-radius: 12px;
  background: #f9f9f9;
}

#pageNotFound a {
  color: black;
  font-size: 18px;
  text-decoration: none;
}

#pageNotFound a:hover {
  text-decoration: underline;
}
