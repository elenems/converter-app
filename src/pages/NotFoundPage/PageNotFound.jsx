import React from "react";
import "./PageNotFound.css";
import { Link } from "react-router-dom";
export default function PageNotFound() {
  return (
    <div id="pageNotFound">
      <div>
        <h1>Page not found</h1>
        <Link to="/">Return home</Link>
      </div>
    </div>
  );
}
