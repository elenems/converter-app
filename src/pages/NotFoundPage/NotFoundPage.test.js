import NotFoundPage from '../NotFoundPage';
import {shallow} from 'enzyme';
import React from 'react';

describe("NotFoundPage", ()=>{
    it('renders', ()=>{
        const wrapper = shallow(<NotFoundPage />).debug();
        expect(wrapper).toMatchSnapshot();
    })
})