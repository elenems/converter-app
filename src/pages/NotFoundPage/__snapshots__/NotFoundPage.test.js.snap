// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders 1`] = `
"<div id=\\"pageNotFound\\">
  <div>
    <h1>
      Page not found
    </h1>
    <Link to=\\"/\\">
      Return home
    </Link>
  </div>
</div>"
`;
